test-src/manageworkertype_test.js

```diff
@@ -17,15 +17,17 @@ describe('provisioner worker type api', () => {
   var workerTypeChanged = _.clone(workerTypeDefinition);
   workerTypeChanged.maxCapacity = 15;
 
-  it('should be able to create a worker (idempotent)', async () => {
+  // Test is disabled for automated testing because it interacts with aws
```

Comment:
yes, because of the ami check stuff I merged last week.

I'd rather figure out how to get AWS credentials into travis than have this method not run.  This is one of the more important test cases here.  Maybe we could instead use sinon to replace the check-for-ami.js function with `() => true`